crates/dpapi/src/lib.rs

```diff
@@ -9,4 +12,4 @@ pub mod rpc;
 pub(crate) mod sid;
 pub(crate) mod str;
 
-pub use error::*;
+pub use error::{DpapiResult, Error};
```

Comment:
note: Naming is a bit inconsistent.

`DpapiResult` / `Error` instead of 1) `DpapiResult` / `DpapiError` or 2) `Result` / `Error`.

Both 1) and 2) are equally good, but I strongly suggest to prefix with the crate name (without importing) for 2): `dpapi::Result` / `dpapi::Error`

Reply:
I prefer the second option :)
done